examples/applications/restapi/handlers.py

```diff
@@ -302,35 +302,55 @@ async def create_tx(self, request):
         - Should handle any kind of output script.( see bitcoinx.address for
         utilities for building p2pkh, multisig etc outputs as hex strings.)
         """
+        frozen_utxos = []
+        account = None
+        tx = None
         try:
             tx, account, password = await self._create_tx_helper(request)
             self.raise_for_duplicate_tx(tx)
             account.sign_transaction(tx, password)
 
-            _frozen_utxos = self.app_state.app.get_and_set_frozen_utxos_for_tx(tx, account)
+            frozen_utxos = self.app_state.app.get_and_set_frozen_utxos_for_tx(tx, account)
```

Comment:
Sign transaction should be atomic in the sense that it should change the utxo/stxo state. If this is the case, what is the gain in freezing utxos?

Reply:
Freezing utxos is an old relic at this point... quite happy to purge utxo freezing from the restapi with your blessing?